widgets/flood/description.js

```diff
@@ -0,0 +1,34 @@
+var R = require('react');
+var DataTable = require('../dataTable');
+var D = R.DOM;
+
+var Description = R.createClass({
+  displayName: 'flood-description',
+  propTypes: {
+    floodData: R.PropTypes.object,
+    isAtRisk: R.PropTypes.bool
+  },
+
+  render: function() {
+    var fields = [
+      ['dataPublishedOn', 'Data published'],
+      ['likelihood.riskCategory', 'Risk category'],
+      ['suitability.scale', 'Suitability scale']
```

Comment:
Ok, how about this?
![screenshot from 2016-02-24 18 53 07](https://cloud.githubusercontent.com/assets/63361/13296959/e668e572-db27-11e5-8939-ed351730d77f.png)